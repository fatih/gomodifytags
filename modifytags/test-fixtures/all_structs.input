package main

type foo struct {
	bar       string
	MyExample bool
	MyAnother []string
}

const exampleVar = "foo"

type bar struct {
	// loose comment

	// home
	ankara string
	yeap   bool // just a boolean

	// great cities
	cities []string

	// seconed loose comment
}
