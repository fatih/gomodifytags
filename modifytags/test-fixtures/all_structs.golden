package main

type foo struct {
	bar       string   `json:"bar"`
	MyExample bool     `json:"my_example"`
	MyAnother []string `json:"my_another"`
}

const exampleVar = "foo"

type bar struct {
	// loose comment

	// home
	ankara string `json:"ankara"`
	yeap   bool   `json:"yeap"` // just a boolean

	// great cities
	cities []string `json:"cities"`

	// seconed loose comment
}
